### ISSUE TITLE
Staging Review finding: No announcement made to screen reader users when pressing "Show all times" button

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#97472]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** CMS Team
**Product:** Next Build and Next Build's Events and Event Listing Templates
**Feature:** Events and Event Listing Templates

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On an event page with other times for the event ([example](https://staging.va.gov/outreach-and-events/events/64661/)), in the "View other times" accordion, a "Show all times" button causes more events to be added to the accordion.

But when events are added, nothing is announced to screen reader users to indicate that a change has occurred. That leaves you wondering if something happened or if the button is broken, and users have to explore the page themselves to determine if it worked or didn't work.

## Recommended action

There are two (ish) ways of handling this:
1. When the button is pressed, send keyboard focus to some other element of the page (with `tabindex="-1"`). Whatever receives focus will be immediately announced.
2. When the button is pressed, populate an `aria-live` region with a status message (eg. "15 events added").

I generally point teams toward option 1 when there's a good status message already on screen that can be announced, eg. a "Showing 35 events" line of text above the list of events --- and I think adding something like that might make sense here. But either that or the `aria-live` option are fine.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by EWashb** [comment]
These issues are already what exists in prod. Fixing them is not within the scope of the Next-Build team. We just replicated current templates. Good finds though!  @laflannery is going to triage these and take them back to the Sitewide teams to address against their own priorities. cc: @briandeconinck

**Comment by shiragoodman** [comment]
Thanks @EWashb - Tagging @artsymartha68 and @humancompanion-usds for their awareness.
